autonomi/src/client/mod.rs

```diff
@@ -266,6 +252,10 @@ impl Client {
 
         client_event_receiver
     }
+
+    pub fn get_evm_network(&self) -> &EvmNetwork {
+        &self.evm_network
+    }
```

Comment:
This is confusing, there is another fn named `get_evm_network` that actually gets it. I suggest we rename this to just `evm_network()` so we know it's not "get"ing it from anywhere

Reply:
Agreed, `get_` is unconventional for getters in Rust too.

Reply:
Modified it!